components/linear_app/linear_app.app.mjs

```diff
@@ -200,5 +213,25 @@ export default {
         },
       };
     },
+    async *paginateResources({ resourcesFn }) {
+      const params = {
+        after: null,
+        first: constants.DEFAULT_LIMIT,
+      };
+      let cursor = true;
```

Comment:
I think `hasNextPage` would be a better name for this variable because `cursor` is usually the hash for the page.

```suggestion
      let hasNextPage = true;
```